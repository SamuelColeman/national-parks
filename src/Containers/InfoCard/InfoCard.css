.info_name_card {
	font-size: 5vh;
	color: #585F29;
}

.info_desc {
	padding: 0 15vh;
	color: #3C321F;
}