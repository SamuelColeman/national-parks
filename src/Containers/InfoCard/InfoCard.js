import React from 'react';
import './InfoCard.css';
import { connect } from 'react-redux';

export const InfoCard = ({ infoName, info }) => {
	if (infoName === 'VisitorCenters') {
		console.log(info)
		return (
			<section>
				<h2>{info.name}</h2>
				<h3>{info.description}</h3>
				<p>{info.url}</p>
			</section>
		)
	} else if (infoName === 'Events') {
		console.log(info)
		return (
			<section>
				<h2>{info.title}</h2>
				<h3>{info.description}</h3>
			</section>
		)
	} else if (infoName === 'Alerts') {
		console.log(info)
		return (
			<section>
				<h2>{info.title}</h2>
				<h3>{info.description}</h3>
			</section>
		)
	} else if (infoName === 'Campgrounds') {
		console.log(info)
		return (
			<section>
				<h2>{info.name}</h2>
				<h3>{info.description}</h3>
			</section>
		)
	}
}

export const mapStateToProps = (state) => ({
  infoName: state.infoName
})

export default connect(mapStateToProps)(InfoCard);