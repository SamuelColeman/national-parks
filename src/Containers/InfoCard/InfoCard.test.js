import React from 'react';
import { shallow } from 'enzyme';
import { InfoCard, mapStateToProps } from './InfoCard';

describe('InfoCard', () => {
	
	let wrapper;

	beforeEach(() => {
    wrapper = shallow(<InfoCard />)
 	})

  it('should match snapshot with correct data passing through', () => {
    expect(wrapper).toMatchSnapshot();
  });
});