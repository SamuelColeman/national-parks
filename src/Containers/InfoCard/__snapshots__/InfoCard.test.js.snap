// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoCard should match snapshot with correct data passing through 1`] = `""`;
